projects/ideabox-groups.md

```diff
@@ -0,0 +1,210 @@
+---
+title: IdeaBox Groups
+---
+
+Every developer has more ideas than time. As David Allen likes to say "the human brain is for creating
+ideas, not remembering them." In this project, we'll be building a simple application for recording and
+archiving our ideas (good and bad alike).
+
+Throughout the project, one of our focuses will be on providing a fluid and responsive client-side
+interface. To this end, we'll rely on JavaScript and jQuery to implement snappy filtering in the
```

Comment:
Do we still want to mention jQuery or should we leave that out for now since it's not covered in Mod 1 anymore?

Reply:
Oh good eye - need to get rid of it.